frontend/app/routes/_gcweb-app.personal-information.address.edit.tsx

```diff
@@ -38,8 +38,8 @@ export async function action({ request }: ActionFunctionArgs) {
   const isValidAddress = (val: string) => val && addressValidationService.isValidAddress(val);
 
   const formDataSchema = z.object({
-    homeAddress: z.string().min(1, { message: 'Enter a home address' }).refine(isValidAddress, { message: 'Invalid home address' }),
-    mailingAddress: z.string().min(1, { message: 'Enter a mailing address' }).refine(isValidAddress, { message: 'Invalid mailing address' }),
+    homeAddress: z.string().min(1, { message: 'personal-information:address.edit.error-message.empty-home-address' }).refine(isValidAddress, { message: 'personal-information:address.edit.error-message.invalid-home-address' }),
+    mailingAddress: z.string().min(1, { message: 'personal-information:address.edit.error-message.empty-mailing-address' }).refine(isValidAddress, { message: 'personal-information:address.edit.error-message.invalid-mailing-address' }),
   });
```

Comment:
While this is not *bad*, you could make it a little easier to read by doing something like:

```
{ message: 'empty-home-address' }
```

and then when you render you could do:

```
<InputField errorMessage={`personal-information:address.edit.error-message.${errorMessages.homeAddress}`}/>
<InputField errorMessage={`personal-information:address.edit.error-message.${errorMessages.homeAddress}`}/>
...etc
```

I don't know if it's any more readable, but it will allow you to remove the repeated `personal-information:address.edit.error-message` from all your keys.

Reply:
updated in commit https://github.com/DTS-STN/canadian-dental-care-plan/pull/214/commits/75ab629978d6061c4f6a3c49f2872e325be338af